src/context/LogContext/index.tsx

```diff
@@ -245,24 +271,35 @@ const LogContextProvider: React.FC<LogContextProviderProps> = ({
         cache.clear(index, 0);
       },
       scrollToLine,
+
       setCaseSensitive: (caseSensitive: boolean) => {
         dispatch({ type: "SET_CASE_SENSITIVE", caseSensitive });
+        Cookie.set(CASE_SENSITIVE, caseSensitive.toString(), { expires: 365 });
       },
+      setExpandableRows,
       setFileName: (fileName: string) => {
         dispatch({ type: "SET_FILE_NAME", fileName });
       },
+      setFilterLogic,
       setLogMetadata: (logMetadata: LogMetadata) => {
         dispatch({ type: "SET_LOG_METADATA", logMetadata });
       },
       setSearch: (searchTerm: string) => {
         dispatch({ type: "SET_SEARCH_TERM", searchTerm });
       },
+      setWrap,
       togglePrettyPrint: (v: boolean) => {
         setPrettyPrint(v);
         Cookie.set(PRETTY_PRINT_BOOKMARKS, v.toString(), { expires: 365 });
       },
     }),
     [
+      wrap,
```

Comment:
Actually I have a question. Does `wrap` truly need to be in the URL? If it always rendered solely based on the cookie, it would lead to a more consistent experience for users.

Reply:
I agree as well wrap seems more like a preference than a shareable value. @mvankeulen94  can you confirm?

Reply:
I talked to Maria about it and she said that there wasn't much of a use case for `wrap` being shareable, so we can turn off sharing for it. I'll update this PR with those changes